docs/content/middlewares/http/compress.md

```diff
@@ -214,3 +214,42 @@ http:
   [http.middlewares.test-compress.compress]
     minResponseBodyBytes = 1200
 ```
+
+### `defaultEncoding`
+
+_Optional, Default=""_
+
+`defaultEncoding` specify the default encoding to use if the header `Accept-Encoding` is defined but empty or if there is no supported encoding.
```

Comment:
This feature would break the RFC:
> An Accept-Encoding header field with a field value that is empty implies that the user agent does not want any content coding in response.

https://www.rfc-editor.org/rfc/rfc9110#section-12.5.3-12

The compress middleware should not try to compress if the client doesn't expect it.

Reply:
yes I reverted that.